infra/api/app-config/dev.tf

```diff
@@ -29,5 +29,10 @@ module "dev_config" {
   # https://docs.aws.amazon.com/opensearch-service/latest/developerguide/what-is.html#choosing-version
   search_engine_version = "OpenSearch_2.15"
 
-  service_override_extra_environment_variables = {}
+  service_override_extra_environment_variables = {
+
+    # Login.gov OAuth
+    ENABLE_AUTH_ENDPOINT    = 1
+    LOGIN_FINAL_DESTINATION = "frontend-dev-1739892538.us-east-1.elb.amazonaws.com/api/auth/callback"
```

Comment:
I'm torn on this. I don't want to over engineer, but I would also very much prefer not to be hard coding the ALB URL inside of our terraform configuration.

Reply:
The frontend does this with a SSM parameter that's written in manually. For consistency that's what we should do here as well

Reply:
https://github.com/HHS/simpler-grants-gov/blob/965b08704fa0ef8c9e65acaffe19aeb545fe1676/infra/frontend/app-config/env-config/environment-variables.tf#L38-L41

Reply:
I can make it a secret if desired, this approach likely gets replaced in the next few months as the redirect will instead be determined by who is calling us most likely